hypothesis-python/tests/cover/test_reproduce_failure.py

```diff
@@ -230,3 +218,16 @@ def test_always_fails(data):
             test_always_fails()
 
     assert "@reproduce_failure" not in out.getvalue()
+
+
+@pytest.mark.parametrize(
+    "ps,b",
+    [
+        (PrintSettings.NEVER, False),
+        (PrintSettings.INFER, True),
+        (PrintSettings.ALWAYS, True),
+    ],
+)
+@checks_deprecated_behaviour
+def test_converts_print_settings_to_boolean(ps, b):
+    assert settings(print_blob=ps).print_blob == b
```

Comment:
`is` instead of `==`, to distinguish between integers and booleans :grimacing: (I am not a fan of `0 == False`...)